microservices/first/main.py

```diff
@@ -49,31 +67,39 @@ def get_deployment_info(leader_ID: int) -> tuple[Response, int]:
     else:
         return jsonify({"error": "Deployment not found"}), 404
 
-@app.route('/update/<int:leader_ID>', methods=['POST'])
-def update_deployment(leader_ID: int) -> tuple[Response, int]:
+@app.route('/update/<int:deployment_id>', methods=['POST'])
```

Comment:
Why are we renaming all leader_ID's back to deployment_id's? Is this intentional?

Reply:
Your db_manager and db_schema both implicitly used deployment ID for queries and set up but referred to them as leader ID's. This was done to reconcile the difference, especially as the DBManager methods were set up to update by deployment ID and not leader ID.